lib/hammer_cli_katello/activation_key.rb

```diff
@@ -81,12 +78,10 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       success_message _("Activation key created")
       failure_message _("Could not create the activation key")
 
-      def request_params
-        params = super
-        if params.key?('max_content_hosts') && !params['max_content_hosts'].match(Common::INTEGER_FORMAT)
```

Comment:
I see that we're removing the `Common::INTEGER_FORMAT`. Is the server going to validate integer format rather than hammer now?